amr-wind/utilities/IOManager.cpp

```diff
@@ -101,6 +107,64 @@ void IOManager::initialize_io()
     }
 }
 
+#ifdef AMR_WIND_USE_ASCENT
+static void ascent_pass(
```

Comment:
What is the reason for this `static` keyword here?

Reply:
This function has internal linkage.